tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -44,6 +44,8 @@ sub run {
     $variables{USE_SAPCONF} = get_var('QESAPDEPLOY_USE_SAPCONF', 'false');
     $variables{SSH_KEY_PRIV} = '/root/.ssh/id_rsa';
     $variables{SSH_KEY_PUB} = '/root/.ssh/id_rsa.pub';
+    $variables{REGISTRATION_PLAYBOOK} = get_var('QESAPDEPLOY_REGISTRATION_PLAYBOOK', 'registration');
```

Comment:
Please add a sanitizing of the value, in case someone forgets and passes to the test something like `QESAPDEPLOY_REGISTRATION_PLAYBOOK=registration.yaml`

Reply:
> Please add a sanitizing of the value, in case someone forgets and passes to the test something like `QESAPDEPLOY_REGISTRATION_PLAYBOOK=registration.yaml`

Are you thinking to some perl code to silently remove `.yaml` or validation to `die`/`croak` if content does not complain?

For example one of the VR is about it (`QESAPDEPLOY_REGISTRATION_PLAYBOOK=donalduck`), what happens if user provide wrong name is that test fails at Ansible step with Ansible that report "Missing playbook `registration.yaml.yaml`"

Reply:
I was thinking of `s/\.yaml$//`, but if the error would be as clear as `"Missing playbook registration.yaml.yaml"`, then that's OK too. What I'd like to avoid is debugging sessions on a failing test and reading something like `QESAPDEPLOY_REGISTRATION_PLAYBOOK=registration.yaml` as normal, but missing a clear error with the 2 `.yaml`.